.github/workflows/update-cmake.yml

```diff
@@ -0,0 +1,55 @@
+name: Update CMake
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/update-cmake.yml'
+      - 'scripts/update_cmake_version.py'
+      - 'CMakeUrls.cmake'
+      - 'docs/index.rst'
+      - 'docs/update_cmake_version.rst'
+      - 'README.rst'
+      - 'tests/test_distribution.py'
+      - 'noxfile.py'
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * *'  # "At 06:00 every day."
+
+jobs:
+  update-dependencies:
+    name: Update dependencies
+    if: github.repository_owner == 'scikit-build' || github.event_name != 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: wntrblm/nox@2022.11.21
+    - name: "Run update: bump CMake"
+      id: bump-cmake
+      run: |
+        nox --force-color -s bump
+        echo "version=$(cat .cmake.version)" >> $GITHUB_OUTPUT
+
+    # we use this step to grab a Github App auth token, so that PRs generated by this workflow
+    # run the GHA tests.
+    - uses: tibdex/github-app-token@v1
+      id: generate-token
+      if: github.ref == 'refs/heads/master' && github.repository == 'scikit-build/cmake-python-distributions'
+      with:
+        app_id: ${{ secrets.SCIKIT_BUILD_BOT_APP_ID }}
+        private_key: ${{ secrets.SCIKIT_BUILD_BOT_APP_PRIVATE_KEY }}
+
+    - name: Create Pull Request
+      if: github.ref == 'refs/heads/master' && github.repository == 'scikit-build/cmake-python-distributions'
+      uses: peter-evans/create-pull-request@v5
+      with:
+        commit-message: Update dependencies
+        title: '[Bot] Update to CMake ${{ steps.bump-cmake.outputs.version }}'
+        body: |
+          Created by update_cmake_version.py
+
+          PR generated by "Update CMake" [workflow](https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}).
+        branch: update-cmake-pr
+        committer: "scikit-build-bot[bot] <83877280+scikit-build-bot[bot]@users.noreply.github.com>"
+        author: "scikit-build-bot[bot] <83877280+scikit-build-bot[bot]@users.noreply.github.com>"
```

Comment:
this needs to be updated once a bot user actually exists.

Reply:
We have https://github.com/scikit-build-bot that could be used for that.

Reply:
I created an app at the organization level as users are less appropriate for this.